### PR TITLE
Export prettierrc

### PR DESCRIPTION
**Pull request by chriszarate**
Export a prettierrc that can be used by projects with this package installed. See README updates for usage. This also updates the `prettierrc` for this project and moves to WP Prettier, so there are included formatting changes.

### CODE REVIEW

File: prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	arrowParens: 'avoid',
+	bracketSpacing: true,
+	jsxBracketSameLine: false,
+	parenSpacing: true,
+	printWidth: 100,
```
Comment (robersongomes) [inline prettierrc.js:6]: [MINOR] What are your thoughts on increasing this? Maybe to `150`?
I tested with the side panel open + 2 split tab editors in a 4k resolution. I could write `150` chars on each tab and there is still `~20` chars left on both tab editors. I use a 32'' monitor though.

**Review by robersongomes — APPROVED** [review]
Looks good! 
I only left a minor suggestion.

File: prettierrc.js (hunk 1 above)
Comment (chriszarate) [reply prettierrc.js:6]: I think this setting is less about ergonomics when writing code and more about reviewing it. Someone on a laptop and using GitHub side-by-side diff might like a lower value.

Comment (robersongomes) [reply prettierrc.js:6]: Got it! (Github wraps long lines in the diff view though 🙃)

Comment (chriszarate) [reply prettierrc.js:6]: True, though the wrap is not as nice as an indented line break. I guess we should discuss it with the team and make adjustments.

**Pull request merged**